edx_notifications/digests.py

```diff
@@ -328,14 +353,37 @@ def _send_user_unread_digest(namespace_info, from_timestamp, to_timestamp, user_
         'grouped_user_notifications': notification_groups
     }
 
+    # create the image dictionary to store the
+    # img_path, unique id and title for the image.
+    open_edx_default_logo = dict(title='Open Edx', path=const.NOTIFICATION_OPEN_EDX_DEFAULT_LOGO, cid=str(uuid.uuid4()))
+    branded_default_logo = dict(title='Branded Default Logo', path=const.NOTIFICATION_BRANDED_DEFAULT_LOGO,
+                                cid=str(uuid.uuid4()))
```

Comment:
I know for PDF receipts we had a dual logo, but - I know for McKA - they will not want two logos. Can we just have a single logo that is put up on the header?